src/views/compute/DeployVM.vue

```diff
@@ -1209,6 +1284,34 @@ export default {
         this.loading.isos = false
       })
     },
+    fetchTemplateOvfProperties (templateId) {
+      this.loading.templateOvfProperties = true
+      const params = {
+        id: templateId
+      }
+      api('listTemplateOvfProperties', params).then(json => {
```

Comment:
Note - upstream PR/feature may get rid of this API; and instead return the properties as part of the listTemplates API response cc @DaanHoogland